apps/nextjs-example/src/components/transactionFlows/SingleSigner.tsx

```diff
@@ -111,18 +112,18 @@ export function SingleSigner() {
     }
   };
 
-  const onSignTransactionV2 = async () => {
+  const onSignRawTransaction = async () => {
     if (!account) return;
 
     try {
       const transactionToSign = await aptosClient(
-        network,
+        network
       ).transaction.build.simple({
         sender: account.address,
         data: {
           function: "0x1::coin::transfer",
           typeArguments: [APTOS_COIN],
-          functionArguments: [account.address, 1], // 1 is in Octas
+          functionArguments: [account.address.toString(), 1], // 1 is in Octas
```

Comment:
need to figure out how we can send the bcs address without converting it into string, sdk complains about it

Reply:
mm good question. maybe `AccountAddress` from a different js bundle?
so `instanceof AccountAddress` returns false internally

Reply:
yes maybe, I thought that also. But both example and core use the same version, I'll test it again